src/common/parser-create-error.js

```diff
@@ -5,6 +5,7 @@ function createError(message, loc) {
   const error = new SyntaxError(
     message + " (" + loc.start.line + ":" + loc.start.column + ")"
   );
+  // @ts-ignore - TBD (...)
```

Comment:
This is duplicated from PR #8759 and can be removed if #8759 is merged. It would be ideal to remove the message from 741240103873e6374886138b9b0503a09b8c3319 from the history when merging.